CONTRIBUTING.md

```diff
@@ -65,23 +66,21 @@ pnpm start
    manually and run `pnpm install` (but make sure to specify an exact dependency
    version rather than a range - i.e. don't prefix the version with a caret or a
    tilde).
-1. Run `pnpm up -r @types/node @types/react @types/react-dom @types/jest` to
-   upgrade [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)
-   packages that must remain in sync with the major versions of the libraries
-   they provide types for.
-
-Note that `pnpm` offers multiple solutions for dealing with peer dependency
-version conflicts and other package resolution issues:
-[`pnpm.overrides`](https://pnpm.io/package_json#pnpmoverrides),
-[`pnpm.packageExtensions`](https://pnpm.io/package_json#pnpmpackageextensions)
-[`peerDependenciesMeta`](https://pnpm.io/package_json#peerdependenciesmeta),
-[`.pnpmfile.cjs`](https://pnpm.io/pnpmfile).
-
-`pnpm.overrides` is currently used to force the version of ESLint to the one
-required by `eslint-config-galex`. This is needed because, in the `demo`
-project, `vite-plugin-eslint` depends on an older version of ESLint.
-
-### Workspace dependencies
+
+> If you run into peer dependency warnings and other package resolution issues,
+> note that `pnpm` offers numerous solutions for dealing with them, like
+> [`pnpm.peerDependencyRules.allowedVersions`](https://pnpm.io/package_json#pnpmpeerdependencyrulesallowedversions).
+
+#### [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) packages
+
+The major versions of `@types/*` packages must be aligned with the major
+versions of the packages they provide types for—i.e. `foo@x.y.z` requires
+`@types/foo@^x`.
+
+For convenience, some `@types` packages can be quickly upgraded to their latest
+minor/patch version by running `pnpm up -r`.
```

Comment:
Thought I'd rewrite this a bit as well, notably to shorten the command to just `pnpm up -r`. We only specify `^` on those exact packages, so the result is the same and it's easier to remember.